Prox/Prox/Widgets/ReviewScoreView.swift

```diff
@@ -41,9 +36,9 @@ class ReviewScoreView: UIStackView {
 
         for index in 0..<5 {
             let scoreItem = ReviewScoreItemView()
-            scoreItem.color = color
             scoreItem.backgroundColor = .clear
-            scoreItem.fill = index < score
+            let scoreLeft = self.score - Float(index)
+            setFillScore(score: scoreLeft, forView: scoreItem)
```

Comment:
I thought about that, but then we would be looping through the views twice, whereas here we only loop through them once. For efficiency I prefer it this way.